### DISCUSSION TITLE
Bandwidth limiter

### DISCUSSION THREAD

**Discussion by elhennig** [abraunegg/onedrive#2700] [Q&A] [answered]
In 2.5 RC1 the uploads are executed in parallel. This is great but also impacts the network performance in my local network. Is there a way to limit the bandwidth to be used by the onedrive client or are there any plans to implement such a function?

**Comment by abraunegg** [comment, marked as answer]
@elhennig 
> Is there a way to limit the bandwidth to be used by the onedrive client or are there any plans to implement such a function?

Please read the documentation as this is detailed in both the v2.4.25 and v2.5.0-rc1 documentation.
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#how-to-rate-limit-the-application-to-control-bandwidth-consumed-for-upload--download-operations
* https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-1/docs/usage.md#how-to-rate-limit-the-application-to-control-bandwidth-consumed-for-upload--download-operations
  * https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-1/docs/application-config-options.md#rate_limit

**Reply by elhennig** [reply]
Thanks and sorry that I have missed that hint...
